alumet.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DEFAULT_ALUMET_CONFIG_FILE="/var/lib/alumet/alumet-config.toml"
+
+alumet_bin_name=$(basename "$0")
+path_to_bin=$(temp=$( realpath "$0"  ) && dirname "$temp")
+LIB_ALUMET="${path_to_bin}/../lib/${alumet_bin_name}_bin"
```

Comment:
This isn't a lib, is it? Why is it called lib?
Why is there a `bin` suffix?

Reply:
it's inside the lib file. 
**_bin** is added to differenciate the bash script and the binary

Reply:
Do you call your plates "kitchen_plate" because they are stored in the kitchen ? :eyes: :upside_down_face: